hack/prepare-release.sh

```diff
@@ -56,8 +54,6 @@ while getopts "abg:k:h" opt; do
             ;;
```

Comment:
the flags processed here don't matcthe `getopts` list. `k` is missing for example.

(The usage content is also out of date).